### ISSUE TITLE
Sub navigation text has color contrast issues

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#7862]
## Description

The active / highlight text for sub navigation links on multiple pages have color contrast issues.

### Steps to reproduce

1. Visit https://www.mozilla.org/en-US/firefox/ (Contrast 2.20 - FAIL)

![image](https://user-images.githubusercontent.com/400117/66464128-4a979780-ea76-11e9-9b02-79e592468a00.png)

2. Visit https://www.mozilla.org/en-US/internet-health/ (Contrast: 2.31 - FAIL)

![image](https://user-images.githubusercontent.com/400117/66463880-e4ab1000-ea75-11e9-9791-85b803d8e71d.png)

3. Visit https://www.mozilla.org/en-US/developer/ (Contrast 2.02 - FAIL)

![image](https://user-images.githubusercontent.com/400117/66463969-0dcba080-ea76-11e9-9921-acca99ef8635.png)

### Expected result

The sub navigation text should be easily readable.

### Actual result

The text is hard to read.

### Environment

Any browser.

**Comment by Shivansh2407** [comment]
Can I help on this?

**Comment by alexgibson** [comment]
These pages have all been decomissioned so the bug is no longer valid, closing.
